### PR TITLE
Feature/mssql plugin

### PR DESCRIPTION
**Pull request by ibarbashov**

### CODE REVIEW

File: mssql-plugin/pom.xml [hunk 1]
```diff
@@ -48,6 +48,11 @@
     </dependency>
 
     <!-- test dependencies -->
+    <dependency>
+      <groupId>com.microsoft.sqlserver</groupId>
+      <artifactId>mssql-jdbc</artifactId>
+      <version>7.2.1.jre8</version>
```
Comment (albertshau) [inline mssql-plugin/pom.xml:54]: should be test scope

File: mssql-plugin/src/test/java/co/cask/mssql/SqlServerPluginTestBase.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    setupBatchArtifacts(DATAPIPELINE_ARTIFACT_ID, DataPipelineApp.class);
+
+    addPluginArtifact(NamespaceId.DEFAULT.artifact(JDBC_DRIVER_NAME, "1.0.0"),
+                      DATAPIPELINE_ARTIFACT_ID,
+                      SqlServerSource.class, SqlServerSink.class, DBRecord.class, ETLDBOutputFormat.class,
+                      DataDrivenETLDBInputFormat.class, SqlServerPostAction.class, SqlServerAction.class);
+
+    // add sqlServer 3rd party plugin
+    PluginClass sqlServerDriver = new PluginClass("jdbc", JDBC_DRIVER_NAME, "sql server driver class",
+                                              SQLServerDriver.class.getName(),
```
Comment (albertshau) [inline mssql-plugin/src/test/java/co/cask/mssql/SqlServerPluginTestBase.java]: nit: fix indentation here and next line

**Review by albertshau — COMMENTED** [review]
Minor comments.

How similar are all these different DB tests? Is there any code that can be consolidated?

File: mssql-plugin/pom.xml (hunk 1 above)
Comment (ibarbashov) [reply mssql-plugin/pom.xml:54]: done

File: mssql-plugin/src/test/java/co/cask/mssql/SqlServerPluginTestBase.java (hunk 2 above)
Comment (ibarbashov) [reply mssql-plugin/src/test/java/co/cask/mssql/SqlServerPluginTestBase.java]: done

**Comment by ibarbashov** [comment]
@albertshau All DB specific tests differs only in data types mapping and jdbc driver usage

**Comment by albertshau** [comment]
OK. After we finish adding everything, it would be good to look at how things can be refactored to remove duplicate code.

**Pull request merged**